src/main/scala/edu/ie3/simona/config/SimonaConfig.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * © 2021. TU Dortmund University,
+ * © 2022. TU Dortmund University,
```

Comment:
I'm not familiar with copyrights but often I see something like: 2021 - 2022 
What to you think?

Reply:
Hmm... I'm not sure, if this is really relevant. The `gradle spotlessApply` takes care of the generation. Thereby, I guess we even cannot get the information anymore, when which file has be initiated...

Reply:
It's because `SimonaConfig` keeps getting regenerated by tscfg. After generation, it doesn't have the typical ie3 header, so after applying spotless, a new header is used with the current year